site/app/controllers/admin/AdminGradeableController.php

```diff
@@ -1133,7 +1133,7 @@ private function updateGradeable(Gradeable $gradeable, $details) {
                             $gradeable->setDependsOnPoints(0);
                         }
                         else {
-                            if ($depends_on_points < 0 || $depends_on_points > $temp_gradeable->getDependsOnPoints()) {
+                            if ($depends_on_points < 0 || ($depends_on_points > $temp_gradeable->getDependsOnPoints() && $temp_gradeable->getDependsOnPoints() != null)) {
```

Comment:
It is better to use the strict comparison operators in PHP.

```suggestion
                            if ($depends_on_points < 0 || ($depends_on_points > $temp_gradeable->getDependsOnPoints() && $temp_gradeable->getDependsOnPoints() !== null)) {
```